pkg/rbac/userData.go

```diff
@@ -186,14 +227,15 @@ func (user *UserDataCache) getClusterScopedResources(cache *Cache, ctx context.C
 	return user, user.csrErr
 }
 
-func (user *UserDataCache) userAuthorizedListCSResource(ctx context.Context, authzClient v1.AuthorizationV1Interface,
+func (user *UserDataCache) userAuthorizedListSSAR(ctx context.Context, authzClient v1.AuthorizationV1Interface,
 	apigroup string, kind_plural string) bool {
 	accessCheck := &authz.SelfSubjectAccessReview{
 		Spec: authz.SelfSubjectAccessReviewSpec{
 			ResourceAttributes: &authz.ResourceAttributes{
-				Verb:     "list",
-				Group:    apigroup,
-				Resource: kind_plural,
+				Namespace: "*",
```

Comment:
Do we need this `Namespace: "*" ` Doc says empty "" for all namespace scoped resources .
```
Namespace is the namespace of the action being requested. Currently, there is no distinction between no namespace and all namespaces "" (empty) is defaulted for LocalSubjectAccessReviews "" (empty) is empty for cluster-scoped resources "" (empty) means "all" for namespace scoped resources from a SubjectAccessReview or SelfSubjectAccessReview```
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#resourceattributes-v1-authorization-k8s-io

Reply:
Ok, will remove that. Thanks.

Reply:
Done.